aws/stage/ecs/main.tf

```
@@ -105,3 +113,135 @@ resource "aws_ecs_service" "hello_world" {
     container_port   = data.aws_ssm_parameter.de_port.value
   }
 }
+
+
+
+resource "aws_ecs_task_definition" "user_api" {
+  family                   = "user_api_task"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = var.container_cpu
+  memory                   = var.container_memory
+  execution_role_arn       = aws_iam_role.ecs_task_execution_role.arn
+  task_role_arn            = aws_iam_role.ecs_task_execution_role.arn
+
+  container_definitions = <<DEFINITION
+[
+  {
+    "image": "xaynetci/xayn_discovery_web_service:livedoor",
+    "cpu": ${var.container_cpu},
+    "memory": ${var.container_memory},
+    "name": "user_api_task",
+    "networkMode": "awsvpc",
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-group": "${aws_cloudwatch_log_group.ecs_container_insights.id}",
+        "awslogs-region": "eu-central-1",
+        "awslogs-stream-prefix": "ecs-container"
+      }
+    },
+    "environment": [
+        {
+          "name": "${data.aws_ssm_parameter.de_port.name}",
+          "value": "${data.aws_ssm_parameter.user_api_port.value}"
+        }
+    ],
+    "portMappings": [
+      {
+        "containerPort": ${data.aws_ssm_parameter.user_api_port.value},
+        "hostPort": ${data.aws_ssm_parameter.user_api_port.value}
+      }
+    ]
+  }
+]
+DEFINITION
+  runtime_platform {
+    operating_system_family = "LINUX"
+    cpu_architecture        = "X86_64"
+  }
+}
+
+resource "aws_ecs_service" "user_api" {
+  name            = "ecs-service"
```

Maybe we can call it differently. 
```suggestion
  name            = "ecs-service-user-api"
```

---

Done!